.ci/project-dependencies.yaml

```diff
@@ -0,0 +1,17 @@
+version: "2.1"
```

Comment:
I will remove this file as soon as https://github.com/kiegroup/drools-ansible-rulebook-integration/pull/14 is merged

Reply:
removed `project-dependencies.yaml`